src/internal/modules/billing/controller.js

```diff
@@ -233,7 +233,7 @@ const badge = {
 
 const getBatchType = (type) => {
   const batchType = type.replace(/^\w/, c => c.toUpperCase());
-  return (batchType === 'Two_part_tarrif') ? 'Two-part tarrif' : batchType;
+  return (batchType === 'Two_part_tariff') ? 'Two-part tariff' : batchType;
```

Comment:
If this is coming from the water service, will it be all lowercase, e.g.
```
(batchType === 'two_part_tariff') ...
```

Reply:
Yes, good point! I just noticed the spelling mistake whilst looking for batch statuses. I'll make it lowercase

Reply:
After a further look, the previous line was capitalising the first letter in the batchType, but as we both found this confusing, I have refactored it to make it a bit clearer